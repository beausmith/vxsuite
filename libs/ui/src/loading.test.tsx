import React from 'react';
import { render } from '@testing-library/react';

import { Loading } from './loading';

test('Renders Loading with defaults', () => {
  const { container } = render(<Loading />);
  expect(container.firstChild).toMatchSnapshot();
});

test('Renders Loading fullscreen with tag and label', () => {
  const { container } = render(
    <Loading isFullscreen as="p">
      Printing
    </Loading>
  );
  expect(container.firstChild).toMatchSnapshot();
});
