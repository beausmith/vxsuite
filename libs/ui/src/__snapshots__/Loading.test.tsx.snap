// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders default 1`] = `
<div
  class="sc-gsDJrp bYhHee"
>
  <h1
    aria-label="Loading."
    class="sc-bdvvaa etEPtU"
  >
    Loading
  </h1>
</div>
`;

exports[`renders fullscreen with tag and label 1`] = `
<div
  class="sc-dkPtyc ktJCZe"
>
  <div
    class="sc-gsDJrp bYhHee"
  >
    <p
      aria-label="Printing."
      class="sc-bdvvaa etEPtU"
    >
      Printing
    </p>
  </div>
</div>
`;
